Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -1439,12 +1439,19 @@ bool RenderBox::hasTrimmedMargin(std::optional<MarginTrimType> marginTrimType) c
     if (!isInFlow())
         return false;
 #if ASSERT_ENABLED
+    // We should assert here if this function is called with a layout system and
+    // MarginTrimType combination that is not supported yet (i.e. the layout system
+    // does not set the margin trim rare data bit for that margin)
+
     // containingBlock->isBlockContainer() can return true even if the item is in a RenderFlexibleBox
     // (e.g. buttons) so we should explicitly check that the item is not a flex item to catch block containers here
-    if (auto* containingBlock = this->containingBlock(); containingBlock && !containingBlock->isFlexibleBox() &&  (containingBlock->isBlockContainer() || containingBlock->isRenderGrid())) {
+    auto* containingBlock = this->containingBlock(); 
+    if (containingBlock && !containingBlock->isFlexibleBox() && (containingBlock->isBlockContainer() || containingBlock->isRenderGrid())) {
```

Comment:
I should have noticed this incorrect use of "containingBlock" in the original PR. While it's ok to land as is, at some point (very low priority) this needs a cleanup/stop using containing block outside of BFC.